src/carts/src/carts-service/dynamo_setup.py

```diff
@@ -0,0 +1,61 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import os
+import time
+import boto3
+import requests
+import json
+from server import app
+
+
+
+
+# DynamoDB table names passed via environment
+ddb_table_carts = os.getenv("DDB_TABLE_CARTS")
+
+# Allow DDB endpoint to be overridden to support amazon/dynamodb-local
+ddb_endpoint_override = os.getenv("DDB_ENDPOINT_OVERRIDE")
+running_local = False
+
+dynamo_client = None
+
+def verify_local_ddb_running(endpoint):
```

Comment:
Can you add some docstrings?

Reply:
For most functions more than a couple of lines (unless it's doing something complex in a couple lines). I trust your judgement. The idea is to just provide a simple explanation so contributors can refer to it rather than going through all the code